sample-apps/payment-customization-sample-app/web/frontend/components/CustomizationPageLayout.jsx

```diff
@@ -22,9 +27,7 @@ export function CustomizationPageLayout({ actionProps, loading, children, ...pro
         breadcrumbs={[{ content: "Customizations", url: "/" }]}
```

Comment:
This page confused me a bit 😆 I got there from the back nav after creating a customization. It seems similar to the payment customizations index but slightly different, more like a "customization configurations" index? 🤔

The only action on that page is delete, but on this page you can activate/deactivate/create:

<img width="891" alt="Screenshot 2022-12-15 at 11 09 33 AM" src="https://user-images.githubusercontent.com/1707217/207913676-19e6c24c-1747-4af9-b3fe-b4e65bf9f640.png">